### ISSUE TITLE
Sitespeed unable to mount pwd to docker container. Getting error overview-page.js does not exist, is the path to the file correct?

### ISSUE THREAD

**Issue by TusAuto** [sitespeedio/browsertime#1463]
1. I have .js script  locally. And i am executing sitespeed script using Docker desktop.
2.  when execute below command in bash

docker run --shm-size=1g --rm -v   "${pwd}:/sitespeed.io" sitespeedio/sitespeed.io:15.6.1  -b  chrome overview-page.js --browsertime.headless    --multi   -n 1

3. getting below  error 
overview-page.js does not exist, is the path to the file correct?
4.  I have checked file using ls and even tried to set explicitly using variable. Still getting file not found error.
5.  I sure i am in correct pwd and script is also available.
6. Please help me with this as we need to use sitespeed docker image to remove browser comapatibilty issue.
7. Any open issue with mounting pwd to docker?

**Comment by soulgalore** [comment]
Hi @TusAuto in your example you put the "-character on the wrong place, the correct way is:
`docker run --rm -v "$(pwd)":/sitespeed.io`

**Comment by TusAuto** [comment]
@soulgalore .. tried with correct syntax still issue is same.

**Comment by soulgalore** [comment]
What OS are you using?

**Comment by TusAuto** [comment]
@soulgalore : Its windows 10.

**Comment by soulgalore** [comment]
At least you needed to map the absolute path on Windows, so try to change `"$(pwd)"` to a full absolute path.

**Comment by TusAuto** [comment]
@soulgalore : Absolute path is working locally. Thing is issue with pwd. 
Locally i executed pwd with / slash like :  docker run --shm-size=1g  --rm  -v "/$(pwd)":/sitespeed.io  sitespeedio/sitespeed.io:15.6.1   script/overview-page.js  --multi  --browsertime.headless  -n 1. And its working fine. But same command when executed in git repository . Overview-page script not found error.

And another strange issue is when i have executed above command locally without / before pwd. Its not working.

**Comment by TusAuto** [comment]
[CI_Pipline_Execution.txt](https://github.com/sitespeedio/browsertime/files/5794322/CI_Pipline_Execution.txt)

**Comment by TusAuto** [comment]
@soulgalore : please guide me on above topic. As we are stuck with pipeline implementation.

**Comment by TusAuto** [comment]
@soulgalore after discussion with devops guys , We came to conclusion that , Docker in docker is not supported by our organization runner. It is disabled by organization due to security reason.   So "docker run " command is not supported in CI. Hence we have adapted node instead of docker. Now its working fine in our CI/CD.  We are getting good UI measures. But are using circle/ci node image which give us Node and browsers and Sitespeed we mentioned in package.json. One headache is to sync image and sites with latest chrome version. But we are managing it :)
